packages/migrate/src/index.ts

```diff
@@ -170,8 +180,8 @@ const Migrate = {
     };
   },
 
-  lastCompletedMigration: async function (options) {
-    let Migrations;
+  lastCompletedMigration: async function (options: Config): Promise<number> {
+    let Migrations: any; // I don't think we have a good type for this yet
```

Comment:
Is it still ok to type this with `import type { Migration } from "./Migration"` for now?

Reply:
No, `Migrations` here refers to a Truffle contract abstraction which we don't really have a type for yet.